utils/dataset_utils.py

```
@@ -242,3 +243,78 @@ def systematic_sample(dataframe, size: int):
+            whole_datasets,
+        ):
+            if training_political_leaning:
+                if balance_center_leaning_class:
+                    dataset = dataset.take_balanced_class_sample_by_size(
+                        train_dataset_sample_size,
+                        (
+                            center_leaning_class_train_size_multipliers[
+                                left_out_dataset.name
+                            ]
+                            if left_out_dataset.name
+                            in center_leaning_class_train_size_multipliers.keys()
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)